### DISCUSSION TITLE
Can I just use it like spigot in BC

### DISCUSSION THREAD

**Discussion by xingxing2008** [games647/FastLogin#695] [Q&A] [answered]
I have a BC sub server (spigot) Only For Login . Because My main server is fabric server.
I have authme 5.6 and fastlogin in sub server.I just want to auto reg/login authme than forward to main server
so I want to don't need put it in BC and MYSQL.JUST like a common spigot use it .
Can I do it ? 

Thanks for your help!

**Comment by TuxCoding** [comment, marked as answer]
The auth plugin is responsible for forwarding a player after a successful authentication. You could make an auth lobby where both plugins are installed. FastLogin only need the confirmation on first activation that the force login command were successful. Afterwards, it will always a force login command.
